commit 51f99a9570ea1955916e5ca1e09356d068c04c99
Author: q66 <q66@chimera-linux.org>
Date:   Sun May 14 20:55:58 2023 +0200

    patch out _FILE_OFFSET_BITS junk
    
    This does not matter for musl so we don't really care if one
    build defines it and another does not.

diff --git a/src/gpgme.h.in b/src/gpgme.h.in
index 1d3c144..d10ac58 100644
--- a/src/gpgme.h.in
+++ b/src/gpgme.h.in
@@ -102,18 +102,6 @@ extern "C" {
 #endif
 
 
-/* Check for a matching _FILE_OFFSET_BITS definition.  */
-#if @NEED__FILE_OFFSET_BITS@
-#ifndef _FILE_OFFSET_BITS
-#error GPGME was compiled with _FILE_OFFSET_BITS = @NEED__FILE_OFFSET_BITS@, please see the section "Largefile support (LFS)" in the GPGME manual.
-#else
-#if (_FILE_OFFSET_BITS) != (@NEED__FILE_OFFSET_BITS@)
-#error GPGME was compiled with a different value for _FILE_OFFSET_BITS, namely @NEED__FILE_OFFSET_BITS@, please see the section "Largefile support (LFS)" in the GPGME manual.
-#endif
-#endif
-#endif
-
-
 
 /*
  * Some opaque data types used by GPGME.
